templates/lxd/index.html

```diff
@@ -0,0 +1,350 @@
+    </div>
+    <div class="col-4">
+      <p>
+      {{ image (
+        url="https://assets.ubuntu.com/v1/338124d1-shields-security.svg",
+        alt="",
+        width="172",
+        height="98",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+     </p>
+      <h3 class="p-heading--4">Transfer control</h3>
+      <p>LXD runs unprivileged containers by default - protecting the host system from potential attacks. For virtual machines, LXD uses modern virtual hardware (<a href="https://wiki.libvirt.org/page/Virtio">VirtIO</a>) exclusively. In addition, it utilises UEFI SecureBoot and provides vTPM support.</p>
+      <p>Resource restrictions through cgroups and ulimits, as well as RBAC, are also supported.</p>
+      <p>LXD is also easy to scale — from containers on your laptop to thousands of instances in the data centre.</p>
+    </div>
+  </div>
+</section>
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width u-sv3">
+    <h2>Alternatives for running workloads</h2>
+  </div>
+  <div class="row u-sv3">
+    <div class="col-6">
+      <h3 class="p-heading--4">System containers</h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">To run a full Linux OS inside a container</li>
+        <li class="p-list__item is-ticked">Utilises Kernel of the host</li>
+        <li class="p-list__item is-ticked">Identical performance to bare metal</li>
+      </ul>
+    </div>
+    <div class="col-6">
+      <p class="p-heading--4">Virtual machines</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">For workloads needing a different kernel or OS than the host</li>
+        <li class="p-list__item is-ticked">Legacy free</li>
+        <li class="p-list__item is-ticked">Cloud-like experience</li>
+      </ul>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h3 class="p-heading--4">Clusters for more demanding workloads</h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Up to 50 servers in a unified cluster with 1000s of instances</li>
+        <li class="p-list__item is-ticked">Run containers, VMs, or a combination of the two</li>
+        <li class="p-list__item is-ticked">Same distributed database</li>
+        <li class="p-list__item is-ticked">Managed uniformly</li>
+      </ul>
+      <p>Want to know more? Watch this <a href="https://www.youtube.com/watch?v=nrOR6yaO_MY">deep dive into LXD clustering</a></p>
+    </div>
+    <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/904e5156-LXD+illustration+2.svg",
+        alt="",
+        width="236",
+        height="214",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```
You can use a `u-fixed-width` to achieve the same thing here, unless you are using it for the every white space that you get from using a `row`